variables.tf

```diff
@@ -55,12 +55,32 @@ variable "multinode_vm_subnet" {
   type = string
 }
 
-variable "compute_is_vm" {
-  description = "Should be set to true if VMs are being used for compute nodes"
-  type = bool
+variable "compute_disk_size" {
+  description = "Block storage root disk size for compute nodes in GB. Set to 0 on baremetal to use physical storage."
+  type = number
+  default = 100
```

Comment:
This will use a disk by default, preventing baremetal from working is that what we want?

Reply:
Most clouds don't provide bare metal, so we need to support VMs for portability. On SMS lab we'll probably mostly stick with bare metal as it's more realistic and there's plenty available

Reply:
I've removed the defaults but updated the README to properly explain what's going on